src/client/dfs/dfs.c

```diff
@@ -865,7 +878,7 @@ open_sb(daos_handle_t coh, bool create, dfs_attr_t *attr, daos_handle_t *oh)
 	/** Open SB object */
 	super_oid.lo = RESERVED_LO;
 	super_oid.hi = SB_HI;
-	daos_obj_generate_id(&super_oid, 0, OC_RP_XSF, 0);
+	daos_obj_generate_id(&super_oid, 0, OC_SX, 0);
```

Comment:
This will introduce incompatibility between master & 0.9. We should really review all the oclass and make then consistent once for all.